<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MEditForm</class>
 <widget class="QWidget" name="MEditForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="settingButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>430</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>生成设置</string>
   </property>
  </widget>
  <widget class="QToolButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>430</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>开始生成</string>
   </property>
  </widget>
  <widget class="QToolButton" name="outputButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>430</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>导出文件</string>
   </property>
  </widget>
  <widget class="QToolButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>430</y>
     <width>171</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>重新选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="Info">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>510</y>
     <width>1071</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>info</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>1240</width>
     <height>40</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="compare">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>200</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>对比模式(暂不可用)</string>
   </property>
  </widget>
  <widget class="QLabel" name="oldImg">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string>原图(暂不可用)</string>
   </property>
  </widget>
  <widget class="QLabel" name="newImg">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>421</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string>地图画(暂不可用)

当前版本请导出预览图查看</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
